edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/PrometheusMetricsParser.cs

```diff
@@ -52,7 +51,7 @@ public static IEnumerable<Metric> ParseMetrics(DateTime timeGeneratedUtc, string
                             timeGeneratedUtc,
                             metricName,
                             metricValue,
-                            JsonConvert.SerializeObject(tags));
```

Comment:
why change this?

Reply:
Tags is now a dictionary. I could make this clearer by explicitly stating the type on line 47